composer.json

```diff
@@ -84,9 +84,12 @@
     "require-dev": {
         "drupal/core-dev": "^9.2.0",
         "drupal/devel": "^4.1",
+        "drupal/upgrade_status": "^3.10",
+        "mglaman/phpstan-drupal": "^0.12.13",
+        "palantirnet/drupal-rector": "^0.11.3",
         "phpspec/prophecy-phpunit": "^2.0",
-        "phpunit/phpunit": "^9.5",
```

Comment:
I believe we need these two to run PHPunit in Docksal.

Reply:
PHPUnit comes as part of drupal/core-dev